lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -126,6 +125,13 @@ def output_cop_comments(output_buffer, cfg, cop_name, offense_count)
         output_cop_param_comments(output_buffer, params, default_cfg)
       end
 
+      def support_autocorrect?(cop_name)
+        cop_class = Cop::Cop.registry.find_by_cop_name(cop_name)
+        return false unless (cop_instance = cop_class&.new)
```

Comment:
I think it will read better if it says `unless cop_class` here. Then you won't need `&.` either when you're creating the cop instance.

Reply:
Ah, cool! I updated it.